### PR TITLE
doc(1.8.0, 1.7.3): volume encryption limitation for COS

### PR DESCRIPTION
**Pull request by c3y1huang**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#10049

#### What this PR does / why we need it:

https://github.com/longhorn/longhorn/issues/10049#issuecomment-2562253129

#### Special notes for your reviewer:

`None`

#### Additional documentation or context

`None`

### CODE REVIEW

**Review by derekbit — COMMENTED** [review]
LGTM

~~Can you add to v1.6.4 as well? Thank you.~~

**Review by derekbit — APPROVED** [review]

**Review by innobead — APPROVED** [review]

File: content/docs/1.7.3/advanced-resources/os-distro-specific/container-optimized-os-support.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ Longhorn provides a GKE COS node agent daemonset, which leverages GKE Kubernetes
 
 ## Limitations
 
-In COS clusters, Longhorn currently supports only V1 data volumes.
+- In COS clusters, Longhorn currently supports only V1 data volumes.
+- For volume encryption, if the built-in `cryptosetup` in COS clusters does not support `argon2i` or `argon2id`, use `pbkdf2` instead. For more details, refer to [GitHub issue](https://github.com/longhorn/longhorn/issues/10049).
```
Comment (jillian-maroket) [inline content/docs/1.7.3/advanced-resources/os-distro-specific/container-optimized-os-support.md:80]:
```suggestion
- You must use `pbkdf2` for volume encryption if the `cryptsetup` utility built into your COS cluster does not support `argon2i` or `argon2id`. For more information, see [Issue #10049](https://github.com/longhorn/longhorn/issues/10049).
```

**Review by jillian-maroket — COMMENTED** [review]
I moved the details around to make the sentence read like a limitation.

File: content/docs/1.7.3/advanced-resources/os-distro-specific/container-optimized-os-support.md (hunk 1 above)
Comment (jillian-maroket) [reply content/docs/1.7.3/advanced-resources/os-distro-specific/container-optimized-os-support.md:80]: FYI I changed `cryptosetup` to `cryptsetup`.

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**